EcoSoapBank/Features/Payment/CheckoutViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  CheckoutViewController.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-28.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
```

Comment:
This file can be deleted; it's been replaced with the MakePaymentVC. Must have snuck back in during a merge.

Reply:
Ok, thanks for the heads-up!